test/common/http/filter/ratelimit_test.cc

```diff
@@ -64,6 +30,7 @@ class HttpRateLimitFilterTest : public testing::Test {
         .WillByDefault(Return(true));
     ON_CALL(runtime_.snapshot_, featureEnabled("ratelimit.test_key.http_filter_enabled", 100))
         .WillByDefault(Return(true));
+    ON_CALL(filter_callbacks_, downstreamAddress()).WillByDefault(ReturnRef(EMPTY_STRING));
```

Comment:
Per our discussion, move this into the main mock definition so that it has a default.